src/objects/api/serializers.py

```diff
@@ -43,13 +43,17 @@ class Meta:
             "registrationAt",
             "correctionFor",
             "correctedBy",
+            "createdOn",
+            "modifiedOn",
```

Comment:
In the issue it was only mentioned to expose these via the admin, so let's stick to that for now
```suggestion
```